enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -337,6 +348,7 @@ func (h *executorHandle) Serve(ctx context.Context) error {
 				executorID = registration.GetExecutorId()
 			} else if req.GetEnqueueTaskReservationResponse() != nil {
 				h.handleTaskReservationResponse(req.GetEnqueueTaskReservationResponse())
+				lastWorkTime = time.Now()
```

Comment:
locally, this fixed the backoff timing -

```suggestion
				lastWorkTime = time.Time{}
```

Reply:
Done.

Reply:
(and thanks!)